library/src/main/scala/scala/scalajs/runtime/LinkingInfo.scala

```diff
@@ -15,7 +15,7 @@ package scala.scalajs.runtime
 import scala.scalajs.js
 
 /** Information about link-time configuration of Scala.js. */
-sealed trait LinkingInfo extends js.Object {
+trait LinkingInfo {
```

Comment:
This is a not a binary-compatible change.

We will have to leave this trait as is and deprecate it. Instead, we need to change `scala.scalajs.LinkingInfo`.